Scripts/Build.ps1

```diff
@@ -9,7 +9,7 @@ param (
 
   # Settings
   [switch]$ci = $false,
-  [string]$config = "Release",
+  [string]$config = "Debug",

```

Comment:
This seems like a more natural default here. I've lost count of the number of times I ran `Test.cmd` on the command line and got stale results because I didn't realize it was testing release bits and Visual Studio was building Debug ones.